.sbtopts

```diff
@@ -1,4 +1,6 @@
 -J-Xms512M
 -J-Xmx4096M
+-J-XX:-UseConcMarkSweepGC
 -J-XX:+UseG1GC
--J-XX:+CMSClassUnloadingEnabled
\ No newline at end of file
+-J-XX:MaxGCPauseMillis=200
```

Comment:
This one is here, because I started from `2.5.x.x` branch. Maybe also a good a idea to have it in `master`